tools/src/test/java/com/nedap/archie/adl14/LargeSetOfADL14sTest.java

```diff
@@ -217,5 +221,23 @@ private Archetype parse(Map<String, Exception> exceptions, Map<String, ANTLRPars
         return null;
     }
 
+    File adl4convertedDir = new File("src/test/resources/adl14converted");
```

Comment:
Apart from the variable name needing to be adl14ConvertedDir - why write all these files, and why commit them all?

if committed, needs a clear marking to which ADL 2 version these are converted in the directory structure.

Reply:
I included the generated files because you can run a conversion (with or without source code changes) and see in the Git diffs if the conversion changed anything compared to the previous run.

I had not originally envisaged that there would need to be two variants of the at-code conversion, since the SEC seems to agree on the mapping method within which at000N codes are not renumbered in the upgrade to ADL2 codes. But of course the Nedap system actually contains ADL2 archetypes converted from ADL1.4 archetypes, with the previous rule, i.e. 'add one to all codes'.

How we call these various conversions is a question for SEC, so I'll post there. But I agree that we probably need two output directories to distinguish the two forms of code conversion.